scripts/benchmarks/format-benchmark-record.js

```diff
@@ -127,6 +133,34 @@ const formatDecileGenerator = function(record) {
   };
 };
 
+// Looks in measures-data.json for an existing measureIds and
+// walks through the original, capitalized + removed spaces,
+// and capitalized + underscored spaces versions.
+// If found, returns the measureId from the measures-data.json file.
+// If none are found, return the padded number or non-spaced version
+const formatMeasureId = (measureId) => {
+  const measureIdRemovedSpaces = measureId.replace(/\s+/g, '');
```

Comment:
Are there any cases with both spaces AND underscores?

Reply:
Ya, it's rare but it can happen. I'll write some regex to fuzzy match

Reply:
Rare as it someone would have to generate it and I havent seen it before